config/01-common-collector-defaults.conf

```diff
@@ -11,4 +11,5 @@
 COLLECTOR_PORT=9619
 
 STATUS_DEFAULT_SCHEDD_PRINT_FORMAT_FILE=/usr/share/condor-ce/ce-status.cpf
+STATUS_DEFAULT_STARTD_PRINT_FORMAT_FILE=/usr/share/condor-ce/pilot-status.cpf
```

Comment:
Unless there's a particular reason the admin may want to set it, I'd prefer not.

(FWIW - I consider the /etc version of everything somewhat of a mistake and want to keep as much as reasonable in /usr/share)